// message.proto
syntax = "proto3";

message AllEmbeddingInput {
    repeated string input = 1;
}

message AllEmbeddingTensorPB {
  repeated int64 shape = 1;
  bytes data = 2;
}

message AllEmbeddingOutput {
    AllEmbeddingTensorPB tensor = 1;
}

service AllEmbeddingRpcService {
    rpc decode(AllEmbeddingInput) returns (AllEmbeddingOutput);
}
